sdk/storage/AzureStorageBlob/Source/Data/BlobTransfer+CoreDataClass.swift

```diff
@@ -47,11 +47,10 @@ public class BlobTransfer: NSManagedObject, TransferImpl {
         return Int64(transfers.filter { $0.state != .complete }.count)
     }
 
-    // TODO: Convert this to return a TransferProgress object
-    /// The current progress of the transfer, calculated as the number of completed blocks divided by the total number
-    /// of blocks that comprise the blob transfer.
-    public var progress: Float {
-        return Float(Int64(transfers.count) - incompleteBlocks) / Float(transfers.count)
+    /// The current progress of the transfer, calculated as the number of bytes transferred divided by the total number
```

Comment:
Probably need to update this comment since the value isn't computed anymore - TransferProgress contains both values.

Reply:
Fixed.